dotCMS/src/main/webapp/html/portlet/ext/folders/order_menu.jsp

```diff
@@ -160,7 +161,7 @@ pagePath=<%=pagePath%>
     
                             <div class="buttonRow">
                                 <% if(showSaveButton){%>
-                                    <button dojoType="dijit.form.Button" onClick="savechanges()" iconClass="saveIcon">
+                                    <button dojoType="dijit.form.Button" onClick="savechanges(this)" iconClass="saveIcon">

```

Comment:
I do not understand how this change fix the problem

Reply:
@freddyucv that solves the save button disabling, the cancel functionality fix is on https://github.com/dotCMS/core-web/pull/1742